.changeset/six-trains-switch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-docs/gatsby-theme-docs': patch
```

Comment:
A new but backwards compatible feature is a minor, not a patch. 
https://semver.org/

Reply:
Thanks for the link 👍